### DISCUSSION TITLE
Error in GUI when using a custom classifier

### DISCUSSION THREAD

**Discussion by w-out** [birdnet-team/BirdNET-Analyzer#283] [General]
Whenever I try to run my custom classifier (for which I have used the "append" option) on multiple files, the GUI gives an error. The only way it will work is when I manually remove the double species names (these are the species that I have added in my classifier but are also already in the standard BirdNET model) from the species list that is created during the training process. However, I am afraid that removing these double species names will influence my results, as I get approximately the same results with my appended classifier compared to BirdNET's own model.

**Comment by Josef-Haupt** [comment]
Please use the "Issues" tab for bugs and problems.
I created [one](https://github.com/kahst/BirdNET-Analyzer/issues/284) for your problem
